src/segments/python.go

```diff
@@ -101,3 +95,41 @@ func (p *Python) canUseVenvName(name string) bool {
 	}
 	return true
 }
+
+func (p *Python) pyenvVersion() (string, error) {
+	// Use `pyenv root` instead of $PYENV_ROOT?
+	// Is our Python executable at $PYENV_ROOT/bin/python ?
+	// Should p.env expose command paths?
+	path := p.env.HasCommandPath("python")
+	// TODO:  pyenv-win has this at $PYENV_ROOT/pyenv-win/shims
+	if path == filepath.Join(p.env.Getenv("PYENV_ROOT"), "shims", "python") {
```

Comment:
Reverse the statement so we can return early rather than indent this entirely?

Reply:
inverted.